Makefile.core.mk

```diff
@@ -6,7 +6,7 @@ ifeq ($(TLS_MODE), boring)
 endif
 
 test:
-	RUST_BACKTRACE=1 cargo test --benches --tests --bins $(FEATURES)
+	FEATURES=$(FEATURES) ./scripts/test-with-coverage.sh 
```

Comment:
can we keep this as a special `coverage` target? Most ofthe time if I just want to test I don't want to waste all the effort and complexity of coverage.

And also not run that in presubmit.